xbmc/pvr/guilib/PVRGUIActionsPlayback.cpp

```diff
@@ -162,7 +162,9 @@ bool CPVRGUIActionsPlayback::PlayRecording(const CFileItem& item, bool bCheckRes
     {
       CFileItem* itemToPlay = new CFileItem(recording);
       itemToPlay->SetStartOffset(item.GetStartOffset());
-      CServiceBroker::GetPVRManager().PlaybackState()->StartPlayback(itemToPlay);
+      CServiceBroker::GetPVRManager().PlaybackState()->StartPlayback(
+          itemToPlay, ContentUtils::PlayMode::CHECK_AUTO_PLAY_NEXT_ITEM,
+          PVR_SOURCE::PVR_SOURCE_SWITCH_LIVE);
```

Comment:
So this doesn't read well now.  When playing a recording we pass `PVR_SOURCE::PVR_SOURCE_SWITCH_LIVE`, which doesn't make a huge amount a sense.  Could change the enum to be simpley `PVR_SOURCE_SWITCH` or `DEFAULT` which would read better, thoughts?

Reply:
> So this doesn't read well now.

I agree.

I would go with "DEFAULT".

Reply:
Done